client/src/main/java/io/confluent/kafka/schemaregistry/client/CachedSchemaRegistryClient.java

```diff
@@ -136,21 +135,19 @@ private int getIdFromRegistry(String subject, Schema schema)
   @Override
   public synchronized int register(String subject, Schema schema)
       throws IOException, RestClientException {
-    Map<Schema, Integer> schemaIdMap;
+    final IdentityLRUCache<Schema, Integer> schemaIdMap;
```

Comment:
For symmetry with the statements below, wanna remove `final` here or add it everywhere below?